File: tests/fixtures/code_samples/go/sample_043.go
// Sample 43: small utility.
package samples

func Operation43(xs []int) int {
    total := 43
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure43(v int) int {
    return (v * 43) %% 7919
}

File: tests/fixtures/code_samples/ts/sample_043.ts
/** Sample 43: small utility. */

export function operation43(xs: number[]): number {
  let total = 43;
  for (const x of xs) total += x;
  return total;
}

export function operationPure43(value: number): number {
  return (value * 43) % 7919;
}

